tests/Issues/Issue3291/config3291.yaml

```diff
@@ -0,0 +1,3 @@
+parameters:
+    sets:
+        - "dead-code"
```

Comment:
Better use just the single rule if you it's 1 rule. It's easier to debug and maintain later.

Reply:
is rule and sets the same in this context? 

Should I replace "deadcode" with "Rector\DeadCode\Rector\Property\RemoveUnusedPrivatePropertyRector" ? 

Not sure I get you.

Reply:
Sets is a set of rules. Here we test just 1 rule - `RemoveUnusedPrivatePropertyRector`.

So it can be registered as a service.

```yaml
services:
    FullNamespace...\RemoveUnusedPrivatePropertyRector: null
```

Reply:
Check this: https://github.com/rectorphp/rector#b-standalone-rules